the-collab-lab/tcl-45-smart-shopping-list
# Mn initial styling

## Description
This PR starts off our styling! Thank you, @N-Zubko, for making such lovely Figma wireframes to help outline the styling!

This also includes a fix that I forgot in my last PR, updating the alert when a user logs out.

## Related Issue
Closes #33 

## Acceptance Criteria
- input base colors into the `App` component: var(--blue), var(--yellow), var(--white) are our go-to!
- input base fonts into the `App` component: 'Share Tech Mono' & 'VT323' & 'Major Mono Display' (if we need it)
- input character images (red, yellow, blue, pink blinky characters & the cute ms. pac-man)
- updated alert for logging out
- styled home page in full! 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|    | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |
|   ✓  | 💅 : Style              |

## Updates

### Before
![Home page before updates](https://user-images.githubusercontent.com/69729289/186303196-f067fc21-1dd2-4b27-aa25-9ab15ce04d2c.png)


### After
![Home page after updates](https://user-images.githubusercontent.com/69729289/186303103-8a4e2148-463c-436a-a574-5bdc79b677cd.png)

![Updated alert message](https://user-images.githubusercontent.com/69729289/186303588-8651ee91-6675-4a50-8c68-06a9b523a06a.png)


## Testing Steps / QA Criteria
1. Checkout `MN-initial-styling` for the latest changes on your local computer and `git pull`, then `npm start`.
2. Enter a list name that doesn't yet exist to see the error message.
3. Enter an existing list name ('brad juicy punk' works, which is hilarious)
4. See the `List` page (not styled!)
5. Click 'log out' 
6. See the updated alert message.
